src/main/java/com/minecolonies/core/entity/citizen/EntityCitizen.java

```diff
@@ -1980,4 +1982,10 @@ public void setMaxAir(final int maxAir)
     {
         this.maxAir = maxAir;
     }
+
+    @Override
+    public String getTeamName()
+    {
+        return level().dimension().hashCode() + "_" + citizenColonyHandler.getColonyId();
```

Comment:
why a string which impacts performance(creates a new string obj everytime to compare, needs to compare strings) instead of just the colony ID as int? just do int getTeam()